boards/modalai/voxl2/target/voxl-px4-set-default-parameters.config

```diff
@@ -95,6 +95,16 @@ param set VOXL_ESC_BAUD 2000000
 param set VOXL_ESC_RPM_MAX 10500
 param set VOXL_ESC_RPM_MIN 1000
 
+# Set the Voxl2 IO outputs to function as motors
+param set VOXL2_IO_FUNC1 101
+param set VOXL2_IO_FUNC2 102
+param set VOXL2_IO_FUNC3 103
+param set VOXL2_IO_FUNC4 104
+
+param set VOXL2_IO_BAUD 921600
+param set VOXL2_IO_MIN 1000
+param set VOXL2_IO_MAX 2000
+
```

Comment:
Why are these VOXL2_IO parameter settings needed for high RPM?

Reply:
These three params are not strictly necessary for high RPM. I had included these because not having these params loaded + not having defaults for these params makes the Voxl2 IO section in the actuator tab look like the image below if the voxl2_io driver is never started... not necessary I suppose but the addition of these params seems rather benign.
![image](https://github.com/modalai/px4-firmware/assets/108781254/2c759136-4736-460d-ae04-b44a06a3a13d)

Reply:
Okay, but fixes for other issues shouldn't be in a single PR. This should only be for high RPM support. There should be another to fix the actuators tab, another to fix this, etc.

Reply:
Ah okay I understand, I will remove this change. Would you like me to make a separate PR for this or leave it be?

Reply:
No, just leave it in. I was just saying that in the future that's the way it needs to be done.